src/test/java/teammates/test/cases/common/SanitizerTest.java

```diff
@@ -111,7 +111,13 @@ private void sanitizeHtml_receivesSanitized_returnsUnchanged() {
     
     @Test
     public void testSanitizeForRichText() {
-        // Not tested - using org.apache.commons.lang3.StringEscapeUtils.escapeHtml4()
+        String actualRichText = "<a href=\"https://teammatesv4.appspot.com\" onclick=\"alert('fail');\"></a>"
+                              + "<script>alert('fail');</script>"
+                              + "<p align=\"center\"><strong>Content</strong></p>";
+        String expectedRichText = "<a href=\"https://teammatesv4.appspot.com\"></a>"
+                                + "<p align=\"center\"><strong>Content</strong></p>";
+        String sanitized = Sanitizer.sanitizeForRichText(actualRichText);
+        assertEquals(expectedRichText, sanitized);
```

Comment:
Two things here:
- The `+`s should be 2 spaces to the right.
- Add more test cases? In particular, try `null` input, empty string, whitespaces, strings that will stay the same after sanitized, and perhaps 1-2 more examples of how the sanitization works (`onclick` and `script` are demonstrated here, let's demo 1-2 other attributes that will be sanitized).